test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
I don't see why that means we wouldn't use an existing suites of tests... especially a suite that already exists, already needs to be run anyway and is run using the ginkgo tech we're using here.